src/qt/optionsmodel.h

```diff
@@ -16,6 +17,9 @@ class Node;
 extern const char *DEFAULT_GUI_PROXY_HOST;
 static constexpr unsigned short DEFAULT_GUI_PROXY_PORT = 9050;
 
+static inline uint64_t PruneGBtoMiB(unsigned int gb) { return (gb * GB_BYTES) >> 20; }
+static inline unsigned int PruneMiBtoGB(uint64_t mib) { return ((mib << 20) + GB_BYTES / 2) / GB_BYTES; }
```

Comment:
Agree with rounding and types. Going to fix.

Do you think the rounding rationale should be commented? If so, what could you suggest?

Reply:
> Do you think the rounding rationale should be commented? If so, what could you suggest?

Maybe something like "Round up when converting from configured MiB to displayed GB to avoid underestimating max disk usage. Round down in the other direction so roundtrip GB -> MiB -> GB conversion is stable."

Reply:
Done.